src/views/CampusSafety/views/LostAndFoundAdmin/views/MissingItemList/index.tsx

```diff
@@ -266,14 +266,18 @@ const MissingItemList = () => {
                       <Grid item xs={1}></Grid>
                     </Grid>
                   )}
-                  {filteredReports.map((report) =>
+                  {filteredReports.map((report, index) =>
                     isMobile ? (
                       // Mobile Layout
                       <Card
                         key={report.recordID}
-                        className={styles.eventCard}
+                        className={`${styles.eventCard} ${
+                          index % 2 === 0 ? styles.evenRow : styles.oddRow
```

Comment:
So the reason why using nth-child wouldn't work here is because im rendering each report as a seperate card component and the cards dont have the .reportRow class, and even if they did in this case the nth-child selector i dont think would apply correctly because of how React components render. Thats why i decided to use the index given from the map

Reply:
Aight, I'mma try it myself to confirm :)

Reply:
@richmondnao I believe CSS is applied after React components render, so it doesn't matter if it's in a map, or even if it's a separate card.  nth-child basically just looks at the occurrences of a css class in the DOM, and applies to the even or odd occurrences of that, so it isn't affected by whether they are separate elements, if that's what you were worried about 👍